sox.features.js

```diff
@@ -1669,9 +1669,21 @@
           $(this).addClass('expander-arrow-small-show');
           const $that = $(this);
           const id = getIdFromUrl($(this).next().attr('href'));
-          $.get(location.protocol + '//' + sox.site.url + '/posts/' + id + '/body', d => {
-            const div = '<div class="linkedPostsInline-loaded-body-sox">' + d + '</div>';
-            $that.next().after(div);
+          var url = $(this).next().attr('href');
```

Comment:
Please could you change this `var` to a `const` (or `let` but I don't think this is changing, so const might be better!)